### ISSUE TITLE
Battlechat link

### ISSUE THREAD

**Issue by peter-budo** [ninetwozero/com.ninetwozero.bf4intel#191]
If we for some reason do another release we should remove Battlechat link from app.
It is not available for download and we will get rid of crash from it https://mint.splunk.com/dashboard/project/f42265ac/errors/2918878055 😆

**Comment by karllindmark** [comment]
Yup! Good call. One of those things you think of but forge to write down lol
